projects/Push-Based-Custom-Ceiling-Partial-Withdraw-for-EIP7251-MaxEB.md

```diff
@@ -0,0 +1,98 @@
+# Push Based Custom Ceiling Partial Withdrawal for EIP-7251 (MaxEB)
+MaxEB, Withdrawal, Deposit, Staking, BeaconStake, Electra, Consensus, Economics
+
+## Motivation
+
+1.  [EIP-7251](https://eips.ethereum.org/EIPS/eip-7251) improves MaxEB, and is [scoping](https://eips.ethereum.org/EIPS/eip-7600) into [Electra](https://ethereum.github.io/consensus-specs/specs/electra/beacon-chain/) Upgrade ([~2025/Q1](https://x.com/TimBeiko/status/1793684244612407687)), and with the mission to [sustain validator set size growth](https://ethresear.ch/t/sticking-to-8192-signatures-per-slot-post-ssf-how-and-why/17989), and preparing for [SSF](https://ethereum.org/en/roadmap/single-slot-finality/) and [ePBS](https://ethereum.org/en/roadmap/pbs/).
+2.  I have [studied](https://hackmd.io/@georgesheth/HJKkx3NSR) that [EIP-7251](https://eips.ethereum.org/EIPS/eip-7251) has 6 features and explains what they are. Push Based Custom Ceiling Partial Withdrawal is a feature still left for design and implementation, and it is demanded for before or post Electra Upgrade, due to its complexity, more details please see bellow analysis.
+3.  This project aims to design and implement Push Based Custom Ceiling Partial Withdrawal at coding level ([Spec](https://ethereum.github.io/consensus-specs/specs/electra/beacon-chain/) and [Test](https://github.com/ethereum/execution-spec-tests)). In summary,
+
+
+## Project description
+
+1.  **[Staking withdrawals](https://ethereum.org/en/staking/withdrawals/)** refer to transfers of ETH from a validator account on Ethereum's consensus layer (the Beacon Chain), to the execution layer where it can be transacted with. Stakers have strong strong demands to withdraw their ETH in validator's account for various purpose. 
```

Comment:
re
```suggestion
1.  **[Staking withdrawals](https://ethereum.org/en/staking/withdrawals/)** refer to transfers of ETH from a validator account on Ethereum's consensus layer (the Beacon Chain), to the execution layer where it can be transacted with. Stakers need to be able to withdraw their ETH deposited as validator. 
```